tasks/main.yml

```diff
@@ -65,7 +66,7 @@
     dest: "/etc/default/postgres_exporter"
     owner: root
     group: root
-    mode: "u=rw,go=r"
+    mode: "u=rw,go="
```

Comment:
Must be readable by the postgres_exporter process group so this needs to remain as is

Reply:
No, this works the way I changed it. We already run the exporter with this. It is read by systemd on init.
Also, because the file contains the credentials to postgres, it is pretty important to close down the permissions from other users.

Reply:
I accept the file needs to be secured, for upstart this needs to be group {{ postgres_exporter_group }} and group readable. This is because the upstart script is executed as the postgres_exporter_user / group. I'll make a subsequent change to do this for upstart only.
Thanks!